### PR TITLE
Tracks: Add woo version as global track prop.

### PR DESCRIPTION
**Pull request by timmyc**
### All Submissions:

* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Context: pbIJXs-vP-p2

TL;DR; In order to have a more effective way to measure impact of new features, we would like to add the WC Version as a global prop on all track events. This branch implements that. If possibly this would be super cool to get in 4.9, but not the end of the world if it doesn't make it.

### Changelog entry

Tweak: Add Woo Version as global prop in track events

### CODE REVIEW

**Review by ObliviousHarmony — APPROVED** [review]
LGTM!

**Review by roykho — APPROVED** [review]
LGTM! Thanks for the PR!

**Comment by pmcpinto** [comment]
@roykho is this going to be included in 5.1?

**Comment by roykho** [comment]
@pmcpinto - yes, if it is tagged a milestone, that's usually the version we will include it in unless we missed it in the process.

**Comment by pmcpinto** [comment]
👍 thanks for the confirmation.
